### ISSUE TITLE
rulefile handling of subrules

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#221]
I'm now also able to disable certain other "subrules" and streamline everything.

Unfortunately, your recent change only seems to work with `--suppress` from the CLI but not with rulefiles.

Just removing `"oelint.var.suggestedvar.CVE_PRODUCT": ""` still reports `oelint.var.suggestedvar.CVE_PRODUCT` errors. I'm also not able to just set `"oelint.var.suggestedvar.CVE_PRODUCT": "info"` inside the rulefile and use `--noinfo` with the CLI; `oelint.var.suggestedvar.CVE_PRODUCT` infos are still reported. Than I thought to mute `oelint.var.suggestedvar` but enable all "subrules", but still this doesn't work:

```json
"oelint.var.suggestedvar": "info",
"oelint.var.suggestedvar.AUTHOR": "error",
"oelint.var.suggestedvar.BBCLASSEXTEND": "error",
"oelint.var.suggestedvar.BUGTRACKER": "error",
"oelint.var.suggestedvar.SECTION": "error",
"oelint.var.suggestedvar.CVE_PRODUCT": "info"
```

_Originally posted by @skycaptain in https://github.com/priv-kweihmann/oelint-adv/issues/220#issuecomment-853311720_
